### DISCUSSION TITLE
What is these percentages on report

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#4677] [Self hosted]
Really need to know about this : Please explain these percentages and what is the "First Response Time" and "Resolution Time" terms here i have already read the reoprt guidelines but still need more detailed info about this but speacially on percentage things. Thank you.

![Screenshot (148) (1)](https://user-images.githubusercontent.com/46310938/168088518-db2aba1f-33ef-41d4-a138-28db03fc725d.jpg)

**Comment by sojan-official** [comment]
@nithindavid  the percentages denote how your inbox is performing when compared to the previous time block. 
for example, if you are looking at the last 7 days' report. the % gives you a comparison of how to report vs the report from 7 days before that.
